proposals/299-ip-failure-count.txt

```diff
@@ -82,7 +82,7 @@ Ticket: https://trac.torproject.org/projects/tor/ticket/27491
    These values will be stored as 32-bit unsigned integers.
 
    A generic failure of an IP version will add one point to the failure point
-   count values of the particular IP version which failed.
+   count of the particular IP version which failed.
 
    A failure of an IP version from a "no route" error which happens when
```

Comment:
I think we also need to add two for "unsupported protocol", and any other permanent failures,

Reply:
Sounds good.